### ISSUE TITLE
Talk Proposal: Frontend security checklist

### ISSUE THREAD

**Issue by davidmytton** [sfnode/sfnode#170]
**Full name and Bio**
David Mytton is currently building [Arcjet](https://arcjet.com/), a product to help developers protect their apps. He also writes the [console.dev](http://console.dev/) weekly devtools newsletter which reviews the most interesting tools for developers. In addition, David is pursuing a PhD in engineering science & sustainable computing at the University of Oxford. In the past, he co-founded Server Density, a cloud infrastructure monitoring startup which was acquired in 2018 by edge computing platform, StackPath.

**Talk title and abstract (description)**
**Title:** Frontend security checklist: 7 things to remember
**Description:** Accidentally exposing secret environment variables. Incorrect validation and sanitization of user input. Content security policies...frontend applications can easily be overlooked, but they have just as many areas that need proper attention from a security perspective. This talk will walk through a checklist of 7 key things developers should consider to improve the security of their frontend & full stack applications.

**Url to Slides or GitHub Repo**
Will share after.

**Social Media Info (Optional)**
https://davidmytton.blog

**Are you going to be live coding?**
No, but I will walk through code examples of real vulnerabilities and how to fix/prevent them.

**Do you need audio from your computer?**
No

**Comment by christian-bromann** [comment]
Hey @davidmytton 👋  thanks for proposing the talk. Would you be available on Thursday 27th this month?

**Comment by davidmytton** [comment]
hi @christian-bromann - unfortunately I'm not in SF at the end of Jun, but I will be at the end of Jul if you wanted to book it in for then?

**Comment by christian-bromann** [comment]
@davidmytton hey David, it seems that July will be already booked for us, can we schedule your talk for August, 29th?

**Comment by davidmytton** [comment]
hey @christian-bromann do you have a meetup planned this month? I'll be in SF Oct 25-Nov 1.

**Comment by christian-bromann** [comment]
@davidmytton yes, we would love to have you speak on October 31st, would that be possible?

**Comment by davidmytton** [comment]
Yes! Let me know the time/place. I'm at david@arcjet.com if you want to coordinate there.

**Comment by christian-bromann** [comment]
Thanks! The tentative date will be Thursday, Oct 31 at 6:30pm at 660 Market St, San Francisco, CA 94104. I will send you an update once we have the event up.

**Comment by christian-bromann** [comment]
@davidmytton are you available on January 30th?

**Comment by davidmytton** [comment]
I'm not available on Jan 30 I'm afraid. I'm still finalizing my travel plans for the first part of the year, so I'll let you know when might work soon! One day we can make this work 😅

**Comment by christian-bromann** [comment]
Ok, keep me posted, I will leave this open.

**Comment by christian-bromann** [comment]
@davidmytton how about February 27th?

**Comment by christian-bromann** [comment]
@davidmytton how about March 27th?

**Comment by davidmytton** [comment]
hey @christian-bromann, I'll be in SF the week before (Mar 17-21). I could do any night that week except Mar 20. If that doesn't work, I'll let you know if I'm going to extend my stay for the following week.

**Comment by christian-bromann** [comment]
> I'll let you know if I'm going to extend my stay for the following week.

Please do. We generally do the meetup the last Thursday of the month and are less flexible on dates.
